libcontainer/specconv/spec_linux.go

```diff
@@ -616,6 +616,15 @@ func CreateCgroupConfig(opts *CreateOpts, defaultDevs []*devices.Device) (*confi
 					Limit:    l.Limit,
 				})
 			}
+			if len(r.Rdma) > 0 {
```

Comment:
It looks like the check for `len` here is redundant -- the `for` below will be a no-op when `len == 0`.

Reply:
With the current code it is, but the correct code would be

```go
if len(r.Rdma) > 0 {
    c.Resources.Rdma = make(map[string]configs.LinuxRdma, len(r.Rdma))
    for k, v := range r.Rdma {
        c.Resources.Rdma[k] = configs.LinuxRdma{
            ......
```

Reply:
Resolved.